public/app/features/explore/utils/panelsRegistry.ts

```diff
@@ -0,0 +1,9 @@
+import { config } from '@grafana/runtime';
+
+export function getPanelForVisType(visType: string) {
```

Comment:
just a stub for now. This needs to return some common interface. Panels registration also needs some way to pass `visualizationType`. Also some can be just hardcoded. There is some custom logic and UI for known panels like Graph which means they need a wrapper as they are bit different than their Dashboard version. This can also be solved by passing some context information.